cli/type_checker.rs

```diff
@@ -212,15 +208,6 @@ impl TypeChecker {
       ));
     }
 
-    // node built-in specifiers use the @types/node package to determine
-    // types, so inject that now (the caller should do this after the lockfile
-    // has been written)
```

Comment:
This isn't really realistic because someone could do `deno run --check main.ts` and then a dynamic import could load deps which would cause this to be put in the lockfile.